scripts/show-features.py

```diff
@@ -155,41 +169,29 @@ def main(argv=None):
             print(f"{args.function} not a function")
             return -1
 
-    print_features(function_handles, extractor)
-
-    return 0
+    print_function_features(function_handles, extractor)
 
 
-def ida_main():
-    import idc
-
-    import capa.features.extractors.ida.extractor
-
-    function = idc.get_func_attr(idc.here(), idc.FUNCATTR_START)
-    print(f"getting features for current function {hex(function)}")
-
-    extractor = capa.features.extractors.ida.extractor.IdaFeatureExtractor()
+def dynamic_analysis(extractor: DynamicExtractor, args):
+    for feature, addr in extractor.extract_global_features():
+        print(f"global: {format_address(addr)}: {feature}")
 
-    if not function:
+    if not args.process:
         for feature, addr in extractor.extract_file_features():
             print(f"file: {format_address(addr)}: {feature}")
-        return
-
-    function_handles = tuple(extractor.get_functions())
 
-    if function:
-        function_handles = tuple(filter(lambda fh: fh.inner.start_ea == function, function_handles))
+    process_handles = tuple(extractor.get_processes())
 
-        if len(function_handles) == 0:
-            print(f"{hex(function)} not a function")
+    if args.process:
+        process_handles = tuple(filter(lambda ph: ph.inner["name"] == args.process, process_handles))
+        if args.process not in [ph.inner["name"] for ph in args.process]:
+            print(f"{args.process} not a process")
             return -1
 
-    print_features(function_handles, extractor)
-
-    return 0
+    print_process_features(process_handles, extractor)
 
 
-def print_features(functions, extractor: capa.features.extractors.base_extractor.FeatureExtractor):
+def print_function_features(functions, extractor: FeatureExtractor):
```

Comment:
this prints all features though so the name is a bit confusing

Reply:
hmm, my reasoning for the naming was that it's printing features at function scope and below, but I see your point; will try to think of a better name.

Reply:
`print_static_features`?

Reply:
and `print_dynamic_features` below?